src/test/resources/Tests/cases/CaseServiceTest/getCasesOk1/expectedResponse.json

```diff
@@ -1,6 +1,6 @@
 [
   {
-    "courthouse": "SWANSEA",
+    "courthouse": "case_courthouse",
```

Comment:
not sure why you changed courthouse values.

Reply:
After the introduction of the changes in how judges were created, lots of the ways that people were using to create test data was causing issues, and so i changed the tests to point to new methods that worked. So the new methods created this name, so didn't see a problem in changing the test expectation.